lib/commands/test.js

```diff
@@ -7,6 +7,9 @@ const SilentError = require('silent-error');
 const path = require('path');
 const Win = require('../utilities/windows-admin');
 const existsSync = require('exists-sync');
+
+require('express').static.mime.define({ 'application/wasm': ['wasm'] });
```

Comment:
is there a reason why we do this in `lib/commands/test.js`? seems unrelated to me 🤔

Reply:
The file serving that we do is already WASM aware (due to changes in broccoli-middleware), this specific line is ensuring that _testem_'s server also uses the correct mime type.  

tldr; its here because its specifically to support testem...

Reply:
could use a comment though...